README.md

```diff
@@ -45,3 +49,18 @@ Set inside frontend folder
 1.  Validate authentication through the user profile API instead of relying on cookie checking.
 2.  Make the Antd table responsive.
 3.  Use streaming to fetch the latest results of reports instead of employing the polling method.
+
+## Released
```

Comment:
An alternative is to tag each release and use the releases pages on GitHub: https://github.com/rtp-atw/nimble-interview/releases